src/reanimated2/ViewDescriptorsSet.ts

```diff
@@ -34,7 +34,7 @@ export function makeViewDescriptorsSet(): ViewDescriptorsSet {
       });
     },
 
-    remove: (viewTag: number) => {
+    remove: (viewTag: number | null) => {
```

Comment:
We know it's nullable but we don't anything about it. Either we should safeguard against null or actually check the code (probably extremely difficult) if it can pass null here.